components/metrics/BUILD.gn

```diff
@@ -22,11 +25,12 @@ static_library("metrics") {
     "delegating_provider.h",
     "drive_metrics_provider.cc",
     "drive_metrics_provider.h",
-    "drive_metrics_provider_android.cc",
-    "drive_metrics_provider_ios.mm",
-    "drive_metrics_provider_linux.cc",
-    "drive_metrics_provider_mac.mm",
-    "drive_metrics_provider_win.cc",
+    # Cobalt only change, all code should be OS agnostic above SB.
```

Comment:
Here as above ( and the rest of the file), it would be preferable to move those into a `if (use_cobalt_customizations)` or `is_starboard` block rather. 

Your version actually provides a more easy to read diff, but the alternate has several benefits and makes collecting the changes we need to drive upstream eventually easier.


eg at the end of this list declaration a block 
```
if !(is_starboard) {
  sources += [
     all the files we can't/dont want to compile for SB
  ]
}
```

Reply:
prefer `use_cobalt_customizations`